packages/cpp/ArmoniK.Api.Client/header/sessions/SessionsClient.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "sessions_common.pb.h"
+#include "sessions_service.grpc.pb.h"
+
+namespace armonik {
+namespace api {
+namespace client {
+
+/**
+ * Session client wrapper
+ */
+class SessionsClient final {
+public:
+  explicit SessionsClient(std::unique_ptr<armonik::api::grpc::v1::sessions::Sessions::StubInterface> stub);
```

Comment:
You should put the definition in the header to enable inlining:
```suggestion
  explicit SessionsClient(std::unique_ptr<armonik::api::grpc::v1::sessions::Sessions::StubInterface> stub) : stub(std::move(stub)) {}
```